import_test.R

```diff
@@ -0,0 +1,25 @@
+
+# install lib
+
+install.packages("rjson")

```

Comment:
```suggestion
if (!suppressWarnings(require(rjson))) install.packages("rjson") 
```
Only install package if not yet on system.